services/bstats/bstats.tester.js

```diff
@@ -0,0 +1,33 @@
+'use strict'
+
+const Joi = require('joi')
+
+const ServiceTester = require('../service-tester')
```

Comment:
We're trying to use a new shorthand for tester instantiation, so would you replace lines 5-9 with this:

`const t = (module.exports = require('../create-service-tester')())`